src/hooks/useSubscriptionPrice.ts

```diff
@@ -0,0 +1,62 @@
+import {useOnyx} from 'react-native-onyx';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import usePreferredCurrency from './usePreferredCurrency';
+import useSubscriptionPlan from './useSubscriptionPlan';
+
+const SUBSCRIPTION_PRICES = {
```

Comment:
Is it better to move it to CONST.ts?

Reply:
Its values are based on `CONST`, so it's impossible to put it in that object, and I also can't see any other `const` being exported from that file, only types

Reply:
I agree this is not ideal but I'm also not seeing a better way to handle it so I think it is what it is